### PR TITLE
Client Changes

### PR DESCRIPTION
**Pull request by mattminwoolee**
There's quite a lot. Feel free to ask me quick questions over zoom.

### CODE REVIEW

**Comment by avincenthill** [comment]
Recommend formatting commit messages as imperative and following this style guide https://chris.beams.io/posts/git-commit/

File: client/src/Info.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import classNames from 'classnames';
+import styles from '../styles/info.css';
+
+const Info = (props) => {
+  return (
+    <div>
+      <div className={ styles.price }>$257.00</div>
```
Comment (nr431207) [inline client/src/Info.jsx]: I wouldn't hardcode the price, instead add it to the database?

File: client/src/Share.jsx [hunk 2]
```diff
@@ -1,15 +1,41 @@
...
   }
 
   render() {
     return(
-      <div>
-        This is the share section
+      <div className={ styles.share}>
+        <ShareModal show={this.state.showModal} handleClose={this.hideModal}></ShareModal>
+        <span className={ styles.click} onClick={this.showModal}>Share</span>
+        <i className={ classNames({ [styles.iconMail]: true, [styles.icon]: true }) }></i>
```
Comment (avincenthill) [inline client/src/Share.jsx]: Very cool!

File: client/src/Checkout.jsx [hunk 3]
```diff
@@ -12,10 +12,9 @@ class Checkout extends React.Component {
 
   render() {
     return (
-      <div>
+      <div className={ styles.checkout}>
```
Comment (avincenthill) [inline client/src/Checkout.jsx]: Missing trailing space per AirBnB styling

File: client/src/Info.jsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+      <div className={ styles.miniSpacing }></div>
+      <div><img className={ classNames({ [styles.iconPrime]: true, [styles.icon]: true }) }></img> | <b>FREE</b> One-Day</div>
+      <div className={ styles.miniSpacing }></div>
+      <div>Get FREE Delivery <b>TOMORROW</b> if you order within 6 hrs 14 mins and choose this date at checkout.<span className={ styles.details }>Details.</span></div>
```
Comment (avincenthill) [inline client/src/Info.jsx]: Plans to make TOMORROW dynamic?

File: client/styles/addToList.css [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  height: 30px;
+  text-align: left;
+}
+
+:local .arrowButton {
+  border-left: 1px solid black;
+  display: inline-block;
+  background: linear-gradient(white, rgb(228, 228, 228));
+  width: 10%;
+  height: 30px;
```
Comment (avincenthill) [inline client/styles/addToList.css]: Prefer not to hardcode heights or widths px values per responsive design

File: client/styles/checkout.css [hunk 6]
```diff
@@ -1,3 +1,7 @@
-:local(.checkout) {
-  color: blue;
+.checkout {
+  font-family: Arial, sans-serif;
+  width: 244px;
+  line-height: 19px;
+  font-size: 13px;
```
Comment (avincenthill) [inline client/styles/checkout.css]: Use vmin vh or vw

File: test/example.js [hunk 7]
```diff
@@ -0,0 +1,4 @@
+const puppeteer = require('puppeteer');
+
+const pageURL = 'http://localhost:3000/';
```
Comment (avincenthill) [inline test/example.js]: Use process.env.PAGEURL here for easier deployment

**Pull request merged**